src/react/2-intro-to-jsx.md

```diff
@@ -302,13 +301,15 @@ One of the most powerful aspects of JavaScript is that it enables developers to
 
 @highlight 2
 
+Note to things above, all event names are camelcased (onClick, onMouseEnter, etc...).
```

Comment:
All of JavaScript is camelcased. I don't think events deserve a special distinction. Writing js handlers in HTML has been obsolete for a while.

Reply:
I still think calling it out here makes sense. While I doubt many people are writing js handlers in HTML the 'ol fashioned way, I know a ton of people that are coming from Angular (which uses ng-click), or Vue (which uses v-on:click) to write handlers, and some extra reminders that it is camelcased in React would serve them well.

Reply:
In that case I'd include the sentence as a continuation of the description in lieu of as a postscript. If it's important enough to mention, I'd argue it's important enough to not be relegated to note-status.

Reply:
Done